wgpu-hal/src/vulkan/adapter.rs

```diff
@@ -320,7 +320,8 @@ impl PhysicalDeviceFeatures {
             | Df::UNRESTRICTED_INDEX_BUFFER
             | Df::INDIRECT_EXECUTION
             | Df::VIEW_FORMATS
-            | Df::UNRESTRICTED_EXTERNAL_TEXTURE_COPIES;
+            | Df::UNRESTRICTED_EXTERNAL_TEXTURE_COPIES
+            | Df::SURFACE_VIEW_FORMATS;
```

Comment:
We need to set this downlevel flag based on support for `VK_KHR_swapchain_mutable_format`.

```rust
dl_flags.set(
    Df::SURFACE_VIEW_FORMATS,
    caps.supports_extension(vk::KhrSwapchainMutableFormatFn::name()),
);
```

Reply:
@cwfitzgerald the flag still gets set here, even though it gets set/unset later, I think we should remove it for clarity.

Reply:
Oh whoops